Content.Server/Decals/DecalSystem.cs

```diff
@@ -98,6 +140,14 @@ public bool TryAddDecal(string id, EntityCoordinates coordinates, [NotNullWhen(t
 
             rotation ??= Angle.Zero;
             var decal = new Decal(coordinates.Position, id, color, rotation.Value, zIndex, cleanable);
+
+            TryAddDecal(decal, gridId, out uid);
+            return true;
+        }
+
+        public void TryAddDecal(Decal decal, GridId gridId, [NotNull] out uint? uid)
+        {
```

Comment:
This doesn't seem to do any of the validation the other method does i.e.
- Checks the prototype actually exists
- Checks the decal isn't getting placed on space.

Reply:
i'll probably jsut move the validation from that method into this one